### DISCUSSION TITLE
Is it possible to style variable text?

### DISCUSSION THREAD

**Discussion by urbgimtam** [pdfme/pdfme#129] [Q&A]
When generating pdfs with dynamic text values, how can I respect the style formatting of the text (bolds, italics, underlines, etc?

I'm guessing that it would require to change fonts, but what would be the format in a text string?

At the moment I have to remove all html tags because it would print the tags also.

**Comment by hand-dot** [comment]
Hi @urbgimtam .
Sorry.
We can't change the text style. we can change only the font.

**Comment by urbgimtam** [comment]
Thanks anyway.

Bruno Mateus


Kyohei Fukuda ***@***.***> escreveu no dia terça, 17/01/2023
à(s) 09:00:

**Comment by Shahzad6077** [comment]
Hey @urbgimtam, did you find any workaround to make the text bold?

**Comment by gxanshu** [comment]
you can try by using a bold or italic fonts. just import it in fonts and use it on your texts

**Comment by hefekranz** [comment]
@hand-dot 
Will this ultimately not be possible? To have bold/italic Words within Text?

**Reply by hand-dot** [reply]
No, it's not possible at the moment.

We are currently in the stage of brainstorming what kind of use cases [this issue](https://github.com/pdfme/pdfme/issues/564) should support.
Could you share what kind of functionality you're looking for in the issue thread?

**Comment by jonelofficial** [comment]
I need this feature also: like highlighting specific word on a text

**Reply by hand-dot** [reply]
Got it. Could you let us know what kind of functionality you're looking for in [this issue](https://github.com/pdfme/pdfme/issues/564) ?
